.github/workflows/unit_tests.yaml

```diff
@@ -14,8 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - 2.5
           - 2.7
+          - 3.0
```

Comment:
This might need to be a string `'3.0'` as it appears to be translating this into "Ruby 3 latest" and running it against Ruby 3.2.

Reply:
Thanks @ehelms! I forgot to mention earlier, it is passing all the checks now!